package.json

```diff
@@ -66,8 +66,8 @@
     "@types/eslint": "^7.28.0",
     "@types/node": "^14.14.7",
     "@types/prettier": "^2.3.2",
-    "@typescript-eslint/eslint-plugin": "^4.29.2",
-    "@typescript-eslint/parser": "^4.29.2",
+    "@typescript-eslint/eslint-plugin": "^4.31.0",
```

Comment:
I was impressed that updating just these 3 packages makes such a big diff in `package-lock.json`.